docs/user-guide/import-certificates.md

```diff
@@ -1,13 +1,172 @@
-# Configure certificates if you have existing certificates
+# Import and configure an existing certificate 
 
-## Import your PKCS12 certificate
+Zowe makes it possible to import and configure existing certificates. The following certificates are supported by Zowe:
+* PKCS12 certificates
+* JCERACFKS certificates 
+
+Review the procedure to import existing PKCS12 or JCERACFKS certificates as well as how to import a certificate stored in an MVS data set into a Zowe keyring.
+## Import an existing PKCS12 certificate
+
+To import a PKCS12 certificate, it is first necessary to import a certificate authority. Then, you can configure the section `>>>> Certificate setup scenario 2` in `zowe.yaml`. 
+
+For PKCS12 certificate users, specify the following parameters in the `zowe.yaml` file:
+
+* Define `zowe.setup.certificate.pkcs12.import.keystore` if you acquired one or more certificates from another CA, stored them in PKCS12 format, and now want to import the certificate(s) into the Zowe PKCS12 keystore.
```

Comment:
Are any of these parameters optional? If they are required or optional, we should note it.

Reply:
Done.